### ISSUE TITLE
logstash isn't available on executable path

### ISSUE THREAD

**Issue by chris-martin** [NixOS/nixpkgs#12966]
I installed the `logstash` package and then tried running `logstash`

```
> logstash
logstash: command not found
```

It doesn't seem like it's actually getting installed?

**Comment by jgillich** [comment]
I can not reproduce. How did you install it and what does `nixos-version` return?

**Comment by chris-martin** [comment]
I installed by adding it to my configuration:

```
environment.systemPackages = with pkgs; [
    # ...
    logstash
];
```

```
> nixos-version
15.09.git.2090c60M (Dingo)
```

**Comment by grahamc** [comment]
Looks like this has been resolved by #12981.
